src/index.js

```diff
@@ -0,0 +1,6 @@
+import printBoard from './printBoard.js';
+import printForm from './addForm.js';
+import './styles/style.css';
+
+printBoard();
+printForm();
```

Comment:
- [ ] I am not able to see anything other than `Leaderboard` written in the header of the webpage. Please make sure that everything render correctly and matches the given wireframe.

### Your webpage

![Screenshot from 2021-09-01 05-03-23](https://user-images.githubusercontent.com/22434660/131589259-af655fa1-2166-49f1-addc-36fc6e9d2451.png)


### Wireframe

![Screenshot from 2021-09-01 05-05-45](https://user-images.githubusercontent.com/22434660/131589243-920c1079-cc06-4aa2-885f-565d0b559f75.png)